pkgs/modules/typescript-language-server/default.nix

```diff
@@ -6,8 +6,7 @@ let
     nativeBuildInputs = [ pkgs.makeWrapper ];
     postInstall = ''
       wrapProgram "$out/bin/typescript-language-server" \
-        --suffix PATH : ${pkgs.lib.makeBinPath [ nodepkgs.typescript ]} \
-        --add-flags "--tsserver-path ${nodepkgs.typescript}/lib/node_modules/typescript/lib/"
+        --suffix PATH : ${pkgs.lib.makeBinPath [ nodepkgs.typescript ]}
```

Comment:
This doesn't work:
```
[languages]

[languages.javascript]
pattern = "**/{*.js,*.jsx,*.ts,*.tsx,*.json}"

[languages.javascript.languageServer]
start = "/nix/store/vmd368fmaklyyj232fvnrawqffd4zwq4-nodejs-20.3.1/bin/node /nix/store/gx32fpn4758z67wa4hqc4wcd7mqrvl9z-typescript-language-server-3.3.2/bin/.typescript-language-server-wrapped --stdio"
```
Is this a necessary change in this PR? I wonder if we can do this separately for easy of rollbacks.